### ISSUE TITLE
apostrophes in track directory names give 505 errors in the apollo display

### ISSUE THREAD

**Issue by mpoelchau** [NAL-i5K/general_issues#145]
When our files were moved over to the new server, it looks like directory names with spaces in them had apostrophes added, e.g.:

on apollo2-dev
```
[mpoelchau@apollo2-dev tracks]$ ls -l /app/data/other_species/bacole/jbrowse/data/tracks
total 4188
drwxrwxr-x.  2713 mpoelchau i5k    90112 Nov 16  2015  bacole_current_models
drwxrwxr-x.  4187 cchilders users 139264 Apr  1  2016 'Genemark prediction'
drwxrwxr-x.  1098 cchilders users  36864 Apr  1  2016 'JAMg golden transcriptome genes'
drwxrwxr-x.  3427 cchilders users 114688 Mar 25  2016 'JAMg OGS v1'
drwxrwxr-x.  9180 cchilders users 299008 Mar 25  2016 'JAMg transcripts de-novo alignments'
drwxrwxr-x. 20029 cchilders users 651264 Apr  5  2016 'Known Insecta Repeats'
drwxrwxr-x.   330 mpoelchau i5k    12288 Nov 16  2015  NCBI_Annotation_Release_100_cDNA_match
```
vs. apollo:
```
mpoelchau@apollo-node1:~ $ ls -l /app/data/other_species/bacole/jbrowse/data/tracks
total 3148
drwxrwxr-x  2713 mpoelchau i5k     98304 Nov 16  2015 bacole_current_models
drwxrwxr-x  4187 cchilders users  135168 Apr  1  2016 Genemark prediction
drwxrwxr-x  1098 cchilders users   36864 Apr  1  2016 JAMg golden transcriptome genes
drwxrwxr-x  3427 cchilders users  135168 Mar 25  2016 JAMg OGS v1
drwxrwxr-x  9180 cchilders users  286720 Mar 25  2016 JAMg transcripts de-novo alignments
drwxrwxr-x 20029 cchilders users  663552 Apr  5  2016 Known Insecta Repeats
drwxrwxr-x   330 mpoelchau i5k     12288 Nov 16  2015 NCBI_Annotation_Release_100_cDNA_match
```

It looks like the apostrophes break the apollo display - all of the tracks with spaces in the directory names give HTTP 505 errors on apollo2-dev: 

![Screen Shot 2020-10-06 at 1 18 40 PM](https://user-images.githubusercontent.com/5487172/95249806-7c048980-07d6-11eb-94d8-ba13688353e2.png)

I'm guessing the easiest fix would be to find all of the directory names with spaces/apostrophes and remove the apostrophes.

**Comment by g8tor** [comment]
@mpoelchau  That is not the issue. I'm not sure what the issue is but the directories do not have apostrophes. If you cd into the directories you and execute pwd you don't see the apostrophes. They appear to only appear in directory listings. For example:

[root@apollo2-dev tracks]# cd Genemark\ prediction/
[root@apollo2-dev Genemark prediction]# pwd
/app/data/other_species/bacole/jbrowse/data/tracks/Genemark prediction

**Comment by mpoelchau** [comment]
Looks like this resolved itself with the nginx config change in #122 ! Thanks @g8tor .
